lib/bokken/accounts/ninja.ex

```diff
@@ -46,8 +46,21 @@ defmodule Bokken.Accounts.Ninja do
     |> cast_embed(:socials, with: &Social.changeset/2)
     |> cast_attachments(attrs, @attachment_fields, allow_urls: true)
     |> validate_required(@required_fields)
+    |> validate_birthday()
     |> assoc_constraint(:guardian)
     |> assoc_constraint(:user)
     |> unique_constraint(:user_id)
   end
+
+  defp validate_birthday(
+         %Ecto.Changeset{valid?: true, changes: %{birthday: birthday}} = changeset
+       ) do
+    if birthday > Date.utc_today() do
```

Comment:
Make sure ninja age is between 7 and 17 years

Reply:
I also thought of that, will implement it 🙏🏻